### PR TITLE
Add CLI Command tree

### PR DESCRIPTION
**Pull request by SanojPunchihewa**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: doc/mi-cli-command-tree.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+remote update [nick-name] [host] [port]
+remote select [nick-name]
+remote show
+```
+
+#### set
+```
+set loglevel [logger-name] [logging-level]
+set loglevel [logging-level]
+remote show
```
Comment (a5anka) [inline doc/mi-cli-command-tree.md]: Why do we have `remote show` here under set?

Comment (SanojPunchihewa) [reply doc/mi-cli-command-tree.md]: It's a typo fixed in ddcf43c

File: doc/resources/mi-cli-command-tree.mmd [hunk 2]
```diff
@@ -0,0 +1,26 @@
+graph LR
```
Comment (a5anka) [inline doc/resources/mi-cli-command-tree.mmd]: Can we add a comment explaining how to generate the image from this file?

Comment (SanojPunchihewa) [reply doc/resources/mi-cli-command-tree.mmd]: Fixed in fd3ef83

**Pull request merged**